docs/getting-started/configuring-your-repository.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Configuring your repository
+---
+
+# Configuring your repository
+
+Once you've [added your first repository](codacy-quickstart.md), **it's important that you configure it** to best match your use case and preferences. An optimal configuration integrates code analysis seamlessly into your development pipeline while avoiding false positives. This encourages your team to follow best practices.
+
+To configure your repository, follow these steps:
+
+1.  [Ignoring files](#ignoring-files)
+1.  [Configuring code patterns](#configuring-code-patterns)
+1.  [Adding coverage reports](#adding-coverage-reports) (optional)
+
+## 1. Ignoring files {: id="ignoring-files"}
+
+[Ignore files](../repositories-configure/ignoring-files.md) to prevent Codacy from analyzing compiled code, third-party libraries, and any arbitrary collection of files and directories.
```

Comment:
I'm afraid that like this we lose some important arguments to convince the users to perform this step - from the major advantage of avoiding false positives to making sure that developers aren't overwhelmed with issues they don't care about and become discouraged from following best practices.

I would restore the previous information while tying it up better with the existing paragraph:

```suggestion
[Ignore files](../repositories-configure/ignoring-files.md) to prevent Codacy from analyzing compiled code, third-party libraries, and any arbitrary collection of non-essential files to avoid false positives, streamline your quality pipeline, and encourage developers to adopt best practices and keep following them.
```